src/components/topbar/TopMenubar.vue

```diff
@@ -136,3 +127,30 @@ onMounted(() => {
   cursor: default;
 }
 </style>
+
+<style lang="postcss">
+/* Desktop: Custom window styling */
+:root[data-platform='electron'] {
```

Comment:
Why are we restoring the logo change? I don't think that is something we want to ship:
1. Our current logo looks bad at icon scale.
3. On-going new logo design.
4. 3. Taking up extra space on the menu bar with no extra benefit provided.